Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ruby:2.1
+MAINTAINER Max Burnette <mburnet2@illinois.edu>
+
+EXPOSE 3000
+
+# Install dependencies
+RUN apt-get update &&\
+    apt-get install -y sudo curl unzip netcat &&\
+    /bin/mkdir /home/bety
+
+# Install postgres client from officially versioned repos
+RUN sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main" > /etc/apt/sources.list.d/pgdg.list' &&\
+    apt-get install wget ca-certificates &&\
+    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add - &&\
+    apt-get -y update &&\
+    apt-get -y upgrade &&\
+    apt-get install -y postgresql-client-9.4
+
+COPY / /home/bety
+
+RUN cd /home/bety &&\
+    /bin/sed -i "/capybara-webkit/ s/^/# /" Gemfile &&\
+    gem install bundler &&\
```

Comment:
I think you can use  `gem install bundler  --without javascript_testing` instead of the sed command.